node-export-server/server.js

```diff
@@ -224,10 +281,136 @@ exports.validateBbox = function(bbox) {
     return null;
 }
 
+/**
+ * updates job with hash's status
+ * We do this in a couple places so figured a function would reduce writing smame things twice.
+ */
+function jobComplete(hash) {
+    jobs[hash].status = 'complete';
+    console.log(jobs[hash].id + ' complete');
+}
+
+/**
+ * Creates a zip archive of outFile.
+ */
+function zipOutput(hash,output,outFile,outDir,outZip,isFile,format,cb) {
```

Comment:
since now we have two places where we create a zip output i put this (which used to be in the callback of the one hoot convert command) into its own function